docs/topics/debug.rst

```diff
@@ -18,19 +18,22 @@ Consider the following scrapy spider below::
             )
 
         def parse(self, response):
-            # collect `item_urls`
+            # < your code goes here >
+            # which collects `item_urls`, an array of urls
```

Comment:
I think it would be better something like

```python
item_urls = # collects some urls
```

or 

```python
# collect some urls and populate `item_urls` with it
```